akka-docs/rst/scala/code/docs/http/scaladsl/server/directives/BasicDirectivesExamplesSpec.scala

```diff
@@ -183,7 +183,7 @@ class BasicDirectivesExamplesSpec extends RoutingSpec {
   }
   "withSettings-0" in compileOnlySpec {
     //#withSettings-0
-    val special = RoutingSettings(system).copy(fileIODispatcher = "special-io-dispatcher")
+    val special = RoutingSettingsImpl(system).copy(fileIODispatcher = "special-io-dispatcher")
```

Comment:
Looking into the code: this should probably just be renamed to RoutingSettings and changed to a non-case class later—the HTTP DSL is still marked experimental so that we can do this.